cmake/toolchains/common/Ubuntu-MinGW-X-Trusty.cmake

```diff
@@ -1,58 +0,0 @@
-IF(WIN64)
-	INCLUDE(${CMAKE_CURRENT_LIST_DIR}/Win64.cmake)
-ELSE()
-	INCLUDE(${CMAKE_CURRENT_LIST_DIR}/Win32.cmake)
-ENDIF()
-INCLUDE(${CMAKE_CURRENT_LIST_DIR}/WinCrossCompile.cmake)
```

Comment:
I believe this is the last reference to `WinCrossCompile.cmake` and that that file can be safely removed as well.

Reply:
@tresf still used in https://github.com/LMMS/lmms/pull/7327/files#diff-d0114d95ca0834f33ca384fc87514a4c0ebce2a404b6c980f20984770fadd07cR17 AFAIK. Doesn't make too much sense that way though, maybe the content of `WinCrossCompile.cmake` should be just inlined there and the redundancies (first two lines I think) removed?

Reply:
Whoops, good catch.  Yeah, I think that's justifiable.

Reply:
Done, code inlined & repeated part refactored, extraneous files removed.